src/components/filterBox/FilterBoxSelectors.ts

```diff
@@ -0,0 +1,27 @@
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../ReactVapor';
+import {defaultMatchFilter, MatchFilter} from '../../utils/FilterUtils';
+import {IFilterState} from './FilterBoxReducers';
+
+export interface GetFilterTextProps {
+    id: string;
+}
+
+const getFilterText = (state: IReactVaporState, props: GetFilterTextProps): string => {

```

Comment:
are we suppose to follow this standard function signature for selectors?

Reply:
if so, could be nice to add this rule in tsjs about selectors

Reply:
I like it, only use the minimal object from the props instead of the full ownProps. avoid a problem with the interface late if we need this selector with an other component